### DISCUSSION TITLE
API return Null response for some locations

### DISCUSSION THREAD

**Discussion by long-nguyen** [weather-gov/api#549] [Q&A]
Some locations return null results like below:
```
        "gridId": null,
        "gridX": null,
        "gridY": null,
        "forecast": null,
        "forecastHourly": null,
        "forecastGridData": null,
        "observationStations": null,

```
Example:

https://api.weather.gov/points/39.3984,-79.3817
Does anyone know why ?

**Comment by JJKraw** [comment]
Good question. Missing data? The grid forecast is accessible on forecast.weather.gov.

  https://forecast.weather.gov/MapClick.php?lon=-79.3817&lat=39.3984

**Comment by sullynole** [comment]
> Some locations return null results like below:
> 
> ```
>         "gridId": null,
>         "gridX": null,
>         "gridY": null,
>         "forecast": null,
>         "forecastHourly": null,
>         "forecastGridData": null,
>         "observationStations": null,
> ```
> 
> Example:
> 
> https://api.weather.gov/points/39.3984,-79.3817 Does anyone know why ?

This has been corrected for the LWX points.  Please send any additional null returns like this to nco.ops@noaa.gov
